runtime/dict.go

```diff
@@ -624,6 +624,34 @@ func dictKeys(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
 	return toDictUnsafe(args[0]).Keys(f).ToObject(), nil
 }
 
+func dictPop(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
```

Comment:
It'd be nice for *Dict to have a method for popping keys since that functionality can't be replicated atomically from the existing interface. So maybe implement *Dict.PopItem() and have dictPop call `d.PopItem(...)`.